compiler/noirc_evaluator/src/ssa/interpreter/errors.rs

```diff
@@ -22,7 +22,7 @@ pub enum InterpreterError {
         rhs_id: ValueId,
         msg: Option<String>,
     },
-    #[error("constrain {lhs_id} != {rhs_id}{message} failed:\n    {lhs} == {rhs}", message = constraint_message(.msg))]
+    #[error("constrain {lhs_id} != {rhs_id}, {message} failed:\n    {lhs} == {rhs}", message = constraint_message(.msg))]
```

Comment:
```suggestion
    #[error("constrain {lhs_id} != {rhs_id}{message} failed:\n    {lhs} == {rhs}", message = constraint_message(.msg))]
```
I thought I sent a comment on this already 🤔 
`constraint_message` inserts he commands if there is a non-empty message.